templates/definition/meter/senec-home.yaml

```
@@ -43,6 +52,7 @@ render: |
     jq: .ENERGY.GUI_BAT_DATA_POWER | sub("fl_"; "")
     scale: -1
   {{- end }}
+  {{- if ne .version "v2" }}
```

```suggestion
  {{- if eq .version "v3" }}
```

Müssen wir für Rückwärtskompatibilität umdrehen.